test/es6module/rlexe.xml

```diff
@@ -87,7 +87,7 @@
     <default>
       <files>module-namespace.js</files>
       <compile-flags>-ES6Module -Es6ToStringTag -args summary -endargs</compile-flags>
-      <tags>exclude_dynapogo,exclude_drt,exclude_sanitize_address</tags>
+      <tags>exclude_dynapogo,exclude_drt,exclude_sanitize_address,exclude_noicu</tags>
```

Comment:
Why does this test fail with noicu?

Reply:
There are some Unicode names in there that get sorted -- I assumed that was related, but I didn't look into it too much. I can open a bug for it.